### ISSUE TITLE
Paypal OTP field not recognised

### ISSUE THREAD

**Issue by madduck** [passff/passff#462]
# General information.
- Versions
  - Operating system: Debian unstable
  - Browser: Firefox 82.0.3
  - PassFF: 1.10.3
  - Host app: 1.2.1-1-g2bceedd

Status line output: `[14:57:53] otp -> (0) no error message`

OTP Input lines setting: `otp,code,_otp,identifier,otpCode`
OTP Auth field names setting: `otpauth,otpCode`

### Actual behaviour
I log in to Paypal, am presented with the 2FA query, but PassFF doesn't recognise it. This is the HTML signature of the input field:

```
<input type="tel" name="otpCode" value="" class="hasHelp hasError" placeholder="Sicherheitscode" autocomplete="off" data-nemo="twofactorOTPInput" maxlength="6" id="otpCode" aria-describedby="otpCode">
```

On Web Console:

```
15:02:00.923 [PassFF.content] 2 matches for https://www.paypal.com/authflow/twofactor/?returnUri=signin&…m%2Fsignin&mkey=authContext:dc9fbe24e39645dfb5d157bed580e29d util.js:66:14
15:02:00.924 [PassFF.content] Best fit item paypal.com/nz.paypal.com@pobox.madduck.net for https://www.paypal.com/authflow/twofactor/?returnUri=signin&…m%2Fsignin&mkey=authContext:dc9fbe24e39645dfb5d157bed580e29d util.js:66:14
```

### Expected behaviour
I want to see the (P) symbol in the input field.

**Comment by tuxor1337** [comment]
That's because "tel" is not an `otpInputType` in the current implementation:
https://github.com/passff/passff/blob/c3456219d204c7df4b9230ba267da4ece6e8810d/src/modules/page.js#L12

**Comment by madduck** [comment]
Okay, that makes sense, sorry that I didn't spot that myself. Could "tel" be added? Want a pull request?

**Comment by tuxor1337** [comment]
It's already fixed, see 40eb9d974ca60ad560ab93d6021ef812285697a4
